src/resolver.ts

```diff
@@ -0,0 +1,32 @@
+import { IResolver } from 'jolocom-lib/js/didMethods/types'
+import { Identity } from 'jolocom-lib/js/identity/identity'
+import { getResolver } from '@jolocom/stacks-did-resolver/js/resolver'
+import { DidDocument } from 'jolocom-lib/js/identity/didDocument/didDocument'
+import { ErrorCodes } from 'jolocom-lib/js/errors'
+import { PREFIX } from './constant'
+import { StacksNetwork } from '@stacks/network'
+import { DIDDocument } from 'did-resolver'
+
+type Resolve = (did: string) => Promise<DIDDocument>
+
+export class StacksResolver implements IResolver {
+  private resolveImplementation: Resolve
+
+  prefix = PREFIX
+
+  constructor(stacksNetwork?: StacksNetwork) {
+    this.resolveImplementation = (did: string) =>
```

Comment:
This still has the same issue, `getResolver` is called each time `StacksResolver#resolve` is called, thus each time constructing a new resolver object, which I don't think is necessary.... unless I am missing something??